backend/dataall/modules/shares_base/services/share_item_service.py

```diff
@@ -1,6 +1,7 @@
 import logging
 
 from dataall.core.permissions.services.resource_policy_service import ResourcePolicyService
+from dataall.core.stacks.aws.ecs import Ecs
```

Comment:
any need for this import?

Reply:
Removing this and few more imports which are not used